src/LedgerLivePlatformSDK/index.ts

```diff
@@ -125,29 +125,44 @@ export default class LedgerLivePlatformSDK {
   /**
    * Start the exchange process by generating a nonce on Ledger device
    * @param {ExchangeType} exchangeType
-   * @param {string} partnerName
    *
    * @returns {Promise<string>} The nonce of the exchange
    */
-  async initExchange(
-    _exchangeType: ExchangeType,
-    _partnerName: string
+  async startExchange(
+    exchangeType: ExchangeType = ExchangeType.FUND
   ): Promise<string> {
```

Comment:
Isn't it actually returning an `ExchangePayload` type (which is indeed a string after all 🤷‍♂️ ) that is then passed to  `completeExchange` as the `binaryPayload` param?

If not, then I don't really understand what this `startExchange` function returns and how it's then used by the third party. And then again I don't know what the `binaryPayload` used in `completeExchange` is nor where it comes from 🤔 

```suggestion
  ): Promise<ExchangePayload> {
```

Reply:
It returns a device transaction id, a nonce used to salt the signature of the payload in order to prevent replay attacks, I will go over the code and make it a bit more resillient